bin/mkosi

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/bash
+
+exec /usr/bin/env python3 -c '
+import getpass
+import os
+import os.path
+import sys
+
+
+def search_mkosi_path():
+    version = sys.version_info
+    user = os.getenv("SUDO_USER") or getpass.getuser()
+
+    if not user:
+        return
+
+    usersite = os.path.join(
+        os.path.expanduser(f"~{user}"),
+        f".local/lib/python{version[0]}.{version[1]}/site-packages"
+    )
+    print(usersite)
+    editable_install = f"{usersite}/mkosi.egg-link"
+    if os.path.exists(editable_install):
+        with open(editable_install) as f:
+            additional_path = f.readline().strip()
+    else:
+        additional_path = usersite
+
+    return additional_path
+
+    return
+
+
+if __name__ == "__main__":
+    try:
+        from mkosi.__main__ import main as mkosi_main
```

Comment:
This os probably the most awkward thing to implement in bash but if we just prefer local installation over system installation I think we're fine since we can just always set PYTHONPATH before invoking python. That also allows both local and system versions to be installed with the local version being preferred over the system version which is great for development.